Mel 789/final documentation updates (#85)
The-Academic-Observatory/oaebu-workflows: docs/README.md

6) Install documentation dependencies: `pip install -r requirements.txt`
7) Build documentation: `make html`
8) To view the documentation, open `_build\html\index.html`.

## MacOS
1) Install Python 3.8 

> 3) In the terminal, navigate to the `oaebu-workflows\docs` folder. 

Reviewer: ```suggestion
3) In the terminal, navigate to the `oaebu-workflows/docs` folder. 
```

Author: Thanks @aroelo ! Done.

3) Create virtual env, if using conda: `conda create -n <nameofvenv> python=3.8 anaconda`
4) Activate virtual env: `conda activate <nameofvenv>`

> 5) Add docs directory directory to Python path: `PYTHONPATH="${PYTHONPATH}:/Users/278559c/Desktop/oaebu-workflows/docs"` followed by `export PYTHONPATH`

Reviewer: ```suggestion
5) Add docs directory directory to Python path: `PYTHONPATH="${PYTHONPATH}:/Users/MyUser/Documents/oaebu-workflows/docs"` followed by `export PYTHONPATH`. Make sure to replace `MyUser` with the name of your user.
```

6) Install documentation dependencies: `pip install -r requirements.txt`
7) Build documentation: `make html`

> 8) To view the documentation, open `_build\html\index.html`.

Reviewer: ```suggestion
8) To view the documentation, open `_build/html/index.html`.
```
